static/js/resize.js

```diff
@@ -23,7 +23,7 @@ function size_blocks(blocks, usable_height) {
     _.each(blocks, function (block) {
         var ratio = block.real_height / sum_height;
         ratio = confine_to_range(0.05, ratio, 0.85);
```

Comment:
Yep!  You can use `git rebase` and then just do a git force-push to update the PR.  
Check out our [GitHub guide](http://zulip.readthedocs.io/en/latest/git-guide.html) and [commit guidelines](http://zulip.readthedocs.io/en/latest/version-control.html) for more details.